doc/ag.txt

```diff
@@ -138,16 +138,36 @@ window is opened, or what size it is. Example: >
   let g:ag_qhandler="copen 20"
 <
 
+                                                           *g:ag_lmapping_func*
+A custom command used to set key mappings for the location list. Default:
+"call ag#AgSetDefaultMappings('l')". Example: >
+  let g:ag_qhandler="call AgSetMyCustomMappings()"
+<
+
+
+                                                           *g:ag_qmapping_func*
+A custom command used to set key mappings for the quickfix window. Default:
```

Comment:
Why not just use a map, like [ctrlp does](https://github.com/kien/ctrlp.vim/blob/b5d3fe66a58a13d2ff8b6391f4387608496a030f/doc/ctrlp.txt#L506-L544)?